src/postamble_minimal.js

```diff
@@ -92,6 +92,8 @@ var wasmExports;
 var wasmModule;
 #endif
 
+var instantiatePromise;
```

Comment:
How about declaring this in `src/audio_worklet.js`

Also, the `instantiatePromise = ` usage below should be wrapped in `if AUDIO_WORKET` I guess?   Although no code size tests were effected maybe because closure can delete this?  Still best not to generate it in the first place though.. especially with MINIMAL_RUNTIME.

Reply:
I'd prefer to leave it as is since adding more ifdefs is going to make this even more unreadable. Closure/rollup/webpack are all able to get rid of unused variables.  Are people running minimal runtime without some minification step?

Reply:
IIRC the goal of minimal runtime is to have the default output be minimal, not depend on DCE steps

@juj, is that correct?

Reply:
Yes absolutely, it is definitely something that MINIMAL_RUNTIME should achieve: smallest output possible.

I can understand that the #ifdef jungle can look a bit messy, but it is worth it in MINIMAL_RUNTIME philosophy. Also the intent is that we should not rely on CLOSURE_COMPILER to optimize code in the "easy cases" that we can deal with ourselves. This is because not every user is able to utilize closure: it requires adhering to defining a heavier set of internal/external public code boundary with the externs model. So not all users can utilize it.

Note that Closure also has challenges of optimizing away variables at global scope.